browser/android/BUILD.gn

```diff
@@ -5,23 +5,35 @@ source_set("android_browser_process") {
   check_includes = false
 
   sources = [
+    "//brave/browser/sync/brave_sync_devices_android.cc",
```

Comment:
Do not include sources from other directories like this. Either move this file or create a dep and add the dep instead

Reply:
I guess this is copied, but can we get a follow-up for it? We really shouldn't have things like this

Reply:
fixed in 618ac477ab3f2ecaef41b1849d8ed22b7113ab57